build.py

```diff
@@ -70,7 +70,8 @@
        '21.06',      # upstream container
        '1.8.0',      # ORT
        '2021.2.200', # ORT OpenVINO
-       '2021.2')     # Standalone OpenVINO
+       '2021.2',     # Standalone OpenVINO
+       '2.2.8')      # DCGM version
```

Comment:
Need to update on L59

Reply:
@dzier / @deadeyegoodwin since we are using a specific version of DCGM maybe we should add this to the release notes?

Reply:
Good idea. Can you open a ticket as a reminder for this.

Reply:
Done: DLIS-2710

Reply:
@jbkyang-nvi  while you are in this code can you also combine 58 and 59 to just "Standalone OpenVINO version"